backend/src/main/java/com/crm/model/db/CarEntity.java

```diff
@@ -0,0 +1,28 @@
+package com.crm.model.db;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "cars")
```

Comment:
Hibernate needs getters and setters. We can use @Data annotation for this. This will generate Equals and hashcode also

Reply:
I think `@Builder` is also good idea

Reply:
Agree. Also, Hibernate requires no-args constructor - this should be added  using Lombok annotation too. ToString would also be handy, but apparently it is already included in @Data annotation.

Reply:
done